docs/src/content/docs/ref/core-math.mdx

```diff
@@ -59,13 +61,169 @@ abs(-42);       // 42
 abs(-(-(-42))); // 42
 ```
 
+## sign
+
+<Badge text="Available since Tact 1.6" variant="tip" size="medium"/><p/>
+
+```tact
+fun sign(x: Int): Int;
+```
+
+Computes and returns the sign of the [`Int{:tact}`][int] value `x`. Produces $1$ is the `x` is negative, $-1$ if the `x` is positive, and $0$ if the `x` is $0$.
```

Comment:
I think it produces 1 if x is positive, -1 if x is negative ....

Reply:
Yup, but examples were correct :)